### PR TITLE
Use Symfony 4.4 by default

### LINKED ISSUE #1368: Symfony 4.4 is not there
**API Platform version(s) affected**: 2.5.3

**Description**  

In release notes states `Upgrade to Symfony 4.4` but it's not possible to update to 4.4 and actual version is 4.3

**How to reproduce**  

```
$ docker-compose exec php composer info | grep symfony/frame

symfony/framework-bundle            v4.3.9  Symfony FrameworkBundle

$ cat api/composer.json:

    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "4.3.*"
        }
    }
```

### PR DESCRIPTION
**Pull request by Deuchnord**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes #1368 
| License       | MIT
| Doc PR        | _N/A_

Using Symfony 4.4 by default, instead of Symfony 4.3.

### CODE REVIEW

File: api/.env [hunk 1]
```diff
@@ -32,13 +39,6 @@ MERCURE_PUBLISH_URL=http://mercure/.well-known/mercure
...
 
-###> symfony/framework-bundle ###
-APP_ENV=dev
-APP_SECRET=!ChangeMe!
-TRUSTED_PROXIES=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
-TRUSTED_HOSTS='^localhost|api$'
-###< symfony/framework-bundle ###
-
 # API Platform distribution
 MERCURE_SUBSCRIBE_URL=https://localhost:1337/.well-known/mercure
```
Comment (teohhanhui) [inline api/.env]: Can we move these to the top before the recipe sections? Because new recipes would add sections at the bottom.

File: api/.env [hunk 2]
```diff
@@ -11,13 +11,20 @@
...
 #
 # Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
-# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+# https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=!ChangeMe!
+#TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
+TRUSTED_HOSTS='^localhost|api$'
```
Comment (teohhanhui) [inline api/.env]: There should not be quotes here.

https://github.com/api-platform/api-platform/pull/1237#issuecomment-571096739

Comment (teohhanhui) [reply api/.env]: Hmm... Actually, that's weird. It should be okay to have quotes here.

File: api/.env [hunk 3]
```diff
@@ -11,13 +11,24 @@
...
+###< symfony/framework-bundle ###
 
 ###> doctrine/doctrine-bundle ###
 # Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
-# For a MySQL database, use: "mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
-# IMPORTANT: You MUST configure your db driver and server version, either here or in config/packages/doctrine.yaml
+# For a PostgreSQL database, use: "postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11&charset=utf8"
```
Comment (dunglas) [inline api/.env]: This must be reverted?

**Comment by dunglas** [comment]
Thanks @Deuchnord

**Pull request merged**